MonoGame.Framework.Content.Pipeline/Serialization/Compiler/TextureCubeWriter.cs

```diff
@@ -20,6 +20,8 @@ protected internal override void Write(ContentWriter output, TextureCubeContent
             if (!level0.TryGetFormat(out format))
                 throw new Exception("Couldn't get format for TextureCubeContent.");
 
+            format = value.GetActualFormat(format);
```

Comment:
On a related note, if it's possible for an OpenGL driver to support framebuffer sRGB but not texture sRGB - *do any OpenGL experts here know if that can happen?* - then that's awkward. I don't know if we just disable sRGB totally if either type isn't available. Otherwise MG users will have to do some gruntwork to make sure they're creating textures and framebuffers in a consistent colour space, which sounds a bit lower-level than XNA / MG usually aims for.